assets/css/style.css

```diff
@@ -594,12 +599,380 @@ li:hover {
+  }
+}
+
+@media screen and (max-width: 411px){
+  *{
+    margin: -6px;
+  }
+  .row-middle{
+    font-size: 18px;
+    line-height: 1.7;
+  }
+  .w{
+    margin-top: 5px;
+    font-size: 60px;
+  }
+  .li .seoi{
+   
+    font-size: 40px;
+  }
+  .l-1{
+    font-size: 30px; 
+  }
+  
+  .row-button button{
+    font-size: 20px;
+    width: 250px; 
+  }
+  #start{
+    margin-left: 35%;
+  }
+  .footer-index h2{
+    font-size: 4vw;
+    text-align: center;
+    margin-left: 70px;
+    margin-right: 5px;
+  } 
+}
+
+@media screen and (max-width: 414px){
+  *{
+    margin: -6px;
+  }
+  .row-middle{
+    font-size: 18px;
+    line-height: 1.7;
+  }
+  .w{
+    margin-top: 5px;
+    font-size: 60px;
+  }
+  .li .seoi{
+   
+    font-size: 40px;
+  }
+  .l-1{
+    font-size: 30px; 
+  }
+  
+  .row-button button{
+    font-size: 20px;
+    width: 250px; 
+  }
+  #start{
+    margin-left: 80px;
+  }
+  .footer-index h2{
+    font-size: 4vw;
+    text-align: center;
+    margin-left: 70px;
+    margin-right: 5px;
+  }
+}
+
+
+@media screen and (max-width: 600px){
+.row-middle .indexlogo{
+  margin: 0;
+  font-size: 6vw;
+  position: relative;
+  padding:0;}
+
+#description{
+padding: 0;
+margin:0;
+}  
+
+.img{
+  margin: 0%;
+  width:60%;
+  display: block;
+}
+
+.images{
+  margin-left: 0px;
 }
 
+.li {
+  font-size: 20vw;
+}
+/* #start{
+margin:0;
+margin-left: 95px;
+} */
+
+.begin{
+  justify-content: center;
+}
+
+.indexb {
+  background-image: url("../img/4.jpg");
+  background-size: cover;
+  background-attachment: fixed;
+  background-position: 50%;
+  background-repeat: no-repeat;
+}
+
+.footer-index {
+  font-size: 6vw;
+}
+
+.header label {
+  margin: 0;
+}
+.header nav {
+  margin:0;
+}
+
+.row{
+  display: block;
+  margin: 0;
+}
+
+.friends {
+  margin: 64px;
+}
+#form {
+  width: 90%;
+}
+
+.row footer {
+  display: block;
+  font-size: 3vw;
+  margin: 0;
+}
+
+.little-title h1 {
+  font-size: 8vw;
+}
+
+#searchStore {
+  margin: 53px;
+}
+
+#map {
+  width: 90%;
+  margin: 0;
+}
+.container.is-fluid {
+  max-width: none!important;
+  padding-left: 27px;
+  padding-right: 35px;
+  width: 107%;
+}
+}
+@media screen and (max-width: 540px){
+  *{
+    margin: -6px;
+  }
+  #index {
+    margin-top: 96px;
+    margin-right: 76px;
+    text-align: center;
+    font-size: 0px;
+}
+  #description{
+    margin-right: 195px;
+  }
+  .column{
+    padding: 1.75rem;
+  }
+  .row-middle{
+    font-size: 18px;
+    line-height: 1.7;
+  }
+  .w{
+    margin-top: 5px;
+    font-size: 60px;
+  }
+  .li{
+    font-size: 67px;
+  }
+  .li .seoi{
+   
+    font-size: 40px;
+  }
+  .l-1{
+    font-size: 30px; 
+  }
+  
+  .row-button button{
+    font-size: 20px;
+    width: 131px;
+  }
+  #start{
+    margin-left: 35%;
+  }
+  .footer-index h2{
+    font-size: 4vw;
+    text-align: center;
+    margin-left: 70px;
+    margin-right: 5px;
+  } 
+
+  img.images{
+    margin-left: 20%;
+  }
+  .header nav {
+    padding-top: 17px;
+    margin-right: 266px;
```

Comment:
I think this should be 0px.

![image](https://user-images.githubusercontent.com/85147307/137746291-db6b8bb1-da84-4ae6-ac31-f7b4c52a418e.png)